packages/postgres/src/dialect.ts

```diff
@@ -32,14 +32,60 @@ export interface PostgresDialectOptions {
    * as the Sequelize team cannot guarantee its compatibility.
    */
   pgModule?: PgModule;
+
+  /**
+   * The PostgreSQL `standard_conforming_strings` session parameter.
+   * Set to `false` to not set the option.
+   * WARNING: Setting this to false may expose vulnerabilities and is not recommended!
+   *
+   * @default true
+   */
+  standardConformingStrings?: boolean;
+
+  /**
+   * The PostgreSql `client_min_messages` session parameter.
+   * Set explicitly to `false` to not override the database's default.
+   * Redshift does not support this parameter, it is important to set this option
+   * to `false` when connecting to Redshift.
+   *
+   * @default 'warning'
+   */
+  clientMinMessages?: string | boolean;
 }
 
 const DIALECT_OPTION_NAMES = getSynchronizedTypeKeys<PostgresDialectOptions>({
+  clientMinMessages: undefined,
   native: undefined,
   pgModule: undefined,
+  standardConformingStrings: undefined,
 });
 
-export class PostgresDialect extends AbstractDialect<PostgresDialectOptions> {
+const CONNECTION_OPTION_NAMES = getSynchronizedTypeKeys<PostgresConnectionOptions>({
```

Comment:
Placing this comment here in postgres, but it counts for the other dialects as well; is there a way for us to retrieve these from the dialect adapter in the future? Otherwise we'll need to keep maintaining this in the future

Reply:
There isn't, but the list synchronization is enforced by typing and double checking which options we allow or don't allow when they are modified will be a good thing for us as some can conflict with our features